docs/modules/reference/pages/service-assurance/monitors/VmwareMonitor.adoc

```diff
@@ -1,60 +1,66 @@
 
 = VmwareMonitor
 
-This monitor is part of the VMware integration provided in Provisiond and tests the power state of a virtual machine (VM) or a host system (host).
-If the power state of a VM or host is poweredOn, the service is up.
-If the VM or Host is powered off, the service will be marked as down.
-By default, standBy is also considered as down.
-When using link:https://www.vmware.com/content/dam/digitalmarketing/vmware/en/pdf/techpaper/Distributed-Power-Management-vSphere.pdf[Distributed Power Management] (automatic PDF download), you can change the standBy state.
-See the ignoreStandBy parameter.
-Optionally, a service down event can be triggered if there are pending alarms within vCenter that have not been acknowledged or cleared.
-
-This monitor requires hosts be imported with Provisiond and the VMware import.
-{page-component-title} requires network access to VMware vCenter server and each of the monitored hosts.
-The credentials for accessing the status of a VMware entity are collected from the responsible VMware vCenter using the settings from `$OPENNMS_HOME/etc/vmware-config.xml`.
-This config file also gets specific asset fields assigned to an imported virtual machine and/or host system.
-The VMware integration populates the following asset fields via Provisiond:
+This monitor is part of the VMWare integration that is provided by provisiond.
+It tests the power state of a virtual machine or host system.
+If the power state is `poweredOn`, the service is considered to be up; if it is `poweredOff`, the service is marked as down.
+By default, a power state of `standBy` will also mark the service as down.
+
+If you use https://www.vmware.com/content/dam/digitalmarketing/vmware/en/pdf/techpaper/Distributed-Power-Management-vSphere.pdf[Distributed Power Management] (automatic PDF download), you can change the `standBy` state (see the `ignoreStandBy` parameter).
+
+Optionally, a service down event can be triggered if there are pending alarms in vCenter that have not been acknowledged or cleared.
+
+== Import hosts
```

Comment:
Could reference the `reference:provisioning/handlers/vmware.adoc` section that's dedicated to setting up the VMware import process.  Also, that section needs some updating for the v30+ External Requistions UI.